packages/integration-sdk-testing/src/jest.ts

```diff
@@ -224,6 +230,72 @@ export interface ToMatchGraphObjectSchemaParams {
   schema?: GraphObjectSchema;
 }
 
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace jest {
+    interface Matchers<R> {
+      toCreateValidRelationshipsToEntities(entities: Entity[]): R;
+    }
+  }
+}
+
+export function toCreateValidRelationshipsToEntities(
```

Comment:
I'm not feeling the name... 🤔 

1. Looking at [where it's used in Azure integration](https://github.com/JupiterOne/graph-azure/blob/79372d18098ed036105386a417c0c8331c3ea03f/src/steps/resource-manager/authorization/index.test.ts#L236), seems like it should be `expect(mappedRelationships).toTargetEntities([...])`. This would reveal the fact that this does not support other kinds of relationships, which took me some time to realize.

2. I'm unsure whether this matcher is meant to ensure that the `jobState` contains the expected entities, since in the Azure use case, it looks like this:

```ts
    const {
      userEntities,
      groupEntities,
      servicePrincipalEntities,
      roleAssignmentEntities,
    } = await getSetupEntities(configFromEnv);
    expect(mappedGroupRelationships).toCreateValidRelationshipsToEntities(
      groupEntities,
    );
```

3. Though the code is attempting to enforce that there is only one entity in the set to which `targetFilterKeys` would apply, IF the intention is to verify that the set of mapped relationships in the job state target something in the set of job state entities, it would seem reasonable to find multiple matches.

Reply:
Great suggestions @aiwilliams. 

1. I agree with the naming convention, thanks for suggesting.
2. In the case of azure, the target entities do not exist in the jobState. They could, however.
3. Good suggestion. I'll change this to require _at least one_ relationship.

Reply:
I ended up adding the option `enforceSingleTarget` to _allow_ the case for exactly one target.